test/SlowTests/Issues/RavenDB-17068.cs

```diff
@@ -18,9 +18,9 @@
 
 namespace SlowTests.Issues;
 
-public class RavenDB_17068: RavenTestBase
+public sealed class RavenDb17068: RavenDB_17068_Base
```

Comment:
can you fix the name, revert it

Reply:
this will mess up with test history on CI

Reply:
Unintentional, sorry!